src/sentry/uptime/consumers/results_consumer.py

```diff
@@ -293,29 +293,16 @@ def handle_result(self, subscription: UptimeSubscription | None, result: CheckRe
 
         try_check_and_update_regions(subscription, result, subscription_regions)
 
-        project_subscriptions = get_project_subscriptions_for_uptime_subscription(subscription.id)
+        project_subscription = get_project_subscription_for_uptime_subscription(subscription.id)
 
-        cluster = _get_cluster()
-        last_updates: list[str | None] = cluster.mget(
-            build_last_update_key(sub) for sub in project_subscriptions
-        )
+        # Nothing to do if there's an orphaned project subscription
+        if not project_subscription:
+            return
```

Comment:
As a follow up, let's delete the `UptimeSubscription`

Reply:
Just added to this PR